utils/src/ooinstall/install_transactions.py

```diff
@@ -41,6 +44,10 @@ def generate_inventory(hosts):
     if 'OO_INSTALL_STAGE_REGISTRY' in os.environ:
         base_inventory.write('oreg_url=registry.access.stage.redhat.com/openshift3/ose-${component}:${version}\n')
 
+    if any(host.hostname == installer_host for host in hosts):
+        base_inventory.write("ansible_connection=local\n")
+        base_inventory.write("ansible_sudo=no\n")
```

Comment:
If we're going to set `ansible_sudo=no` we should probably add a check that verifies they are running the installer as root.  We'd need a decent error message too like, "It appears you are running the installer on a host where __ will be installed.  Please rerun the installation as root".

I'm definitely concerned some people won't want to run anything as root so we'll likely have to think more about this.  I think where we left off with sudo is that we'd have to enforce passwordless sudo for the quick installer.